Two-body-scf-guess.py

```diff
@@ -111,16 +111,6 @@ def two_body_guess(mol, monomers, dimers, bs):
     dimers = [[(ms[0]+ms[1]), (ms[0]+ms[2]), (ms[0]+ms[3])], [(ms[0]+ms[1]), (ms[1]+ms[2]), (ms[1]+ms[3])], 
               [(ms[0]+ms[2]), (ms[1]+ms[2]), (ms[2]+ms[3])], [(ms[0]+ms[3]), (ms[1]+ms[3]), (ms[2]+ms[3])]]
     
-    monomer_dens = make_monomer_dens(monomers, bs)
-    m,m = monomer_dens[0].shape
-    a = len(monomer_dens)
-    list_of_ddls = make_list_of_ddls(dimers, bs)
-    z_rho = make_z_rho(m, a)
-    ei_list = make_ei_list(list_of_ddls)
-    i_list = fill_ei_list(list_of_ddls, ei_list, m)
-    overlap = add_interacts(i_list, m)
-    diags = compute_diags(overlap, monomer_dens, a)
-    rho_d = rho_diagonal(diags, z_rho, m)
-    rho = rho(list_of_ddls, rho_d, m)
     tbg = two_body_guess(mol, monomers, dimers, bs)
-    print(tbg)
+    nitr = run_scf(mol, bs, tbg)
+    return nitr
```

Comment:
I'd just print `nitr`. If you return it (and it's not 0) the shell is going to take that as an error code. Otherwise add a unit test and I think you're good to go.

Reply:
Just did ryan.